extensions/cornerstone/src/commandsModule.ts

```diff
@@ -19,6 +19,9 @@ import toggleStackImageSync from './utils/stackSync/toggleStackImageSync';
 import { getFirstAnnotationSelected } from './utils/measurementServiceMappings/utils/selection';
 import getActiveViewportEnabledElement from './utils/getActiveViewportEnabledElement';
 import { CornerstoneServices } from './types';
+import { split } from 'lodash';
+import getSopClassHandlerModule from 'extensions/default/src/getSopClassHandlerModule';
+import { HangingProtocol } from 'platform/core/src/types';
```

Comment:
this is not how you import from core, use `import from @ohif/core`

Reply:
actually there is OhifTypes above imported

Reply:
removed